### DISCUSSION TITLE
Importing wallpapers

### DISCUSSION THREAD

**Discussion by 23fpsz** [rocksdanister/lively#461] [Q&A]
Can i be able to use wallpapers from wallpaper engine in lively wallpaper as there are some amazing wallpapers in wallpaper engine(but it is paid).

**Comment by rocksdanister** [comment]
If its [supported format.](https://github.com/rocksdanister/lively#features)
If its using wpe api then will require [rewrite.](https://github.com/rocksdanister/lively/wiki/API)
